lib/components/page1/page1.component.js

```diff
@@ -4,7 +4,7 @@ import './page1.scss';
 class Page1 {
 	static detailsComponent() {
 		return {
-			templateUrl: Page1View,
+			template: Page1View,
```

Comment:
ngtemplate loader does not seem to play nice with lazy loading.
I switched to HTML loader for the page 1 component.
Switch from templateUrl to simply `template`